web/pandas/about/governance.md

```diff
@@ -49,145 +49,110 @@ manage project donations and acts as a parent legal entity. NumFOCUS is the
+project who made significant contributions in the form of code, reviews,
+software design, documentation etc. Their role in the project is to
+advance the pandas software and goals.
+
+**Membership**: Contributors to the pandas project become maintainers after
+showing significant contributions over a period of one year, and with
+consensus from the rest of activate maintainers
+
+See the list of active maintainers [here](team.html#maintainers).
+
+### BDFL (Benevolent Dictator For Life)
+
+The figure of the BDFL exist to be able to unblock situations where a decision
+needs to be made, and consensus or voting has failed. In such situations, the
+BDFL will make the final decision.
+
+**Membership**: Wes McKinney, as original creator of pandas has been the BDFL
+of the project. In the event of Wes stepping down as BDFL, maintainers will
+make a decision about whether to appoint a new BDFL or change the governance
+type.
+
+### Finances committee
+
+The role of the members of the finances committee is to approve the spending
+of pandas funds. Decisions in general will be made together with the rest of
+active maintainers. Committee members will be responsible to make the final
+decisions, and formally approve payments.
+
+**Membership**: The committee will have 5 members, who will be selected by active
+maintainers. Some constraints exists regarding committee membership:
+
+- Members must be active maintainers
+- No more than two committee members can be employed directly or indirectly
+  by the same employer
+- Committee members should not have conflicts of interest that could prevent
+  them to make the best decisions in the interest of the project. This includes
+  maintainers who receive significant payments from pandas funds
+
+### Code of conduct committee
+
+The role of the committee is to make sure pandas is as open, transparent and
+inclusive as it aims to be by its values. In particular, the committee will
+monitor and respond to any possible violation of our code of conduct. And
+will publish regular summaries about violation reports.
+
+**Membership**: Any members of the community can be part of the committee.
+The committee will have 5 members, who will be selected by active maintainers.
+The next constraints must be satisfied:
+
+- The committee should aim to be as diverse as reasonably possible, to be able
+  to make decisions based on a variety of points of views. In particular, the
+  committee should not have more than 3 members of the same gender, or more
+  than two members from the same geography (continent). Ideally the committee
+  will also be diverse in other ways such as religion, political views,
+  age, etc.
+- No more than two members of the committee should be pandas maintainers.
+
+### Inactive maintainers
+
+Inactive maintainers are former active maintainers who are not interested or
+not available to continue contributing to pandas in a regular way. If they
```

Comment:
You'd have to define "regular way".

We have a way to bring people in, but what defines "regular" ?

Reply:
My intention was to provide more an intuition more than a formal definition in this first part, so people understand what this group is about. And in the membership section, define clearer what are the conditions to become part of this group.

In this sense, it's not very relevant what regular exactly implies. And it'd be up to the affected person to decide if they consider that they stopped making contributions in a regular way, and what regular means to them. And clearly that could mean different things for different people. And a maintainer may decide to be considered inactive after one month, and another consider themselves active after 3 years of no contributions. But to me personally this is fine.

Reply:
I think it's okay in this introductory paragraph to use "regular" as long as the `Membership` section is clear what "regular" means.